os_net_config/cli.py

```diff
@@ -429,27 +444,30 @@ def main(argv=sys.argv, main_logger=None):
         try:
             from os_net_config import dcb_config
         except ImportError as e:
-            logger.error(f'cannot apply DCB configuration: {e!r}')
+            logger.error("cannot apply DCB configuration: %s", str(e))
             return 1
 
         utils.configure_dcb_config_service()
         dcb_apply = dcb_config.DcbApplyConfig()
         dcb_apply.apply()
 
     if purge_provider and migration_failed is False:
-        logger.info('Cleaning the residue files from '
-                    f'{opts.purge_provider} provider')
+        logger.info(
+            "Cleaning thei residue files from %s provider", opts.purge_provider
+        )
         purge_provider.clean_migration()
     elif migration_failed:
-        logger.info('Migration Failed. Reverted back to '
-                    f'{opts.purge_provider} provider')
+        logger.info(
+            "Migration Failed. Reverted back to %s provider",
+            opts.purge_provider,
+        )
         return 1
 
     if opts.noop:
         if configure_sriov:
             files_changed.update(pf_files_changed)
         for location, data in files_changed.items():
-            print("File: %s\n" % location)
+            print("File: %s\n", location)
```

Comment:
This is invalid. print() is not handling %s automatically. I have no idea what this code is supposed to do but you could probably change it to:

```python
print("File:", location)
print()
```